.circleci/config.yml

```diff
@@ -585,7 +585,7 @@ jobs:
     parallelism: 2
     steps:
       - run_test:
-          pattern: 'django$'
+          pattern: 'django($|_celery)'
```

Comment:
@brettlangdon are we happy running these two together or should we have a dedicated job? I went for this approach because a dedicated job for just 1 test felt overkill

Reply:
no, I think running together makes sense to me.